proxy/pkg/middleware/basic_auth.go

```diff
@@ -85,6 +85,7 @@ func BasicAuth(optionSetters ...Option) func(next http.Handler) http.Handler {
 				// fake oidc claims
 				claims := map[string]interface{}{
 					oidc.OwncloudUUID:      user.Id.OpaqueId,
+					options.UserOIDCClaim:  user.Id.OpaqueId,
```

Comment:
Doesn't this give you a dupliacte key here? E.g. when `options.UserOIDCClaim == oidc.Email` (which AFAIU even is the default)

Reply:
It indeed duplicates the key in some cases... But Go is fine with duplicate key as long as one of the duplicate keys is a variable... (https://play.golang.org/p/juVyt9SRnUf).

While looking at this I noticed, that `options.UserOIDCClaim` should only be set if use the `userid` to search for users on the CS3apis.

Reply:
Ah, thanks for clarifying. Didn't know that.